group_project_v2/project_api/dtos.py

```diff
@@ -0,0 +1,82 @@
+from group_project_v2.utils import make_user_caption
+
+
+# pylint:disable=too-many-instance-attributes
+class UserDetails(object):
+    def __init__(self, **kwargs):
+        self.id = kwargs.get('id', None)
+        self.email = kwargs.get('email', None)
+        self.username = kwargs.get('username', None)
+        self.uri = kwargs.get('uri', None)
```

Comment:
The other classes in this file use `url` instead of `uri`, including `ReducedUserDetails`. Can you make them consistent?